File: problems_001to025/problem_015/solution.py
import os
import sys
from math import factorial

PROJECT_PATH = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
sys.path.append(PROJECT_PATH)

from measurement import Stopwatch


class Solution(Stopwatch):
    def __init__(self):
        super().__init__()

    def execute(self):
        answer = factorial(40) // (factorial(20) * factorial(20))

        return answer

if __name__ == '__main__':
    soln = Solution()
    result = soln.execute()
    print(result)

File: problems_001to025/problem_015/solution.ts
import { SolutionFunction } from '@/lib/types';
import { factorial } from '@/lib/toolbox';

const solution: SolutionFunction = () => {
  let answer: ReturnType<SolutionFunction> = null;

  answer = Number(factorial(40) / (factorial(20) * factorial(20)));

  return answer;
};

export default solution;
